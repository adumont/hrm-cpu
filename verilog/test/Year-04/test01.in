15 1e
33 c4
44 a1
39 90
21 83
05 7d
54 36
26 c9
40 1b
24 05
57 fc
19 3c
50 b3
20 de
34 3b
50 44
15 f1
15 b2
04 e3
24 53
56 ef
35 18
23 c6
