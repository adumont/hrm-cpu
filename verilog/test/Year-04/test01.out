c4
1e
90
a1
7d
83
c9
36
05
1b
3c
fc
de
b3
44
3b
b2
f1
53
e3
18
ef
